my-react-app/src/Tabs/Tabs.js

```diff
@@ -1,22 +1,29 @@
-import React from "react";
+import React, { useState } from "react";
+import Mytabs1 from "./Mytabs1";
+import Mytabs from "./Mytabs";
 function Tabs(){
     
-    const items = ['Tab1','Tab2','Tab3'];
+    const [tab, showTab] = useState(1);  
+
+        return (
+          <div className="p-3">
+
+                <button className="p-3"
+                  onClick={() => showTab(0)}
```

Comment:
Create a common function to update the state.

example : 

const showHideTab = (tabIndex) => {
showTab(tabIndex)
}

onClick={() => showHideTab(0)}
onClick={() => showHideTab(1)}